apps/site-landing/src/containers/CreateAccountBanner/index.tsx

```diff
@@ -12,9 +9,9 @@ export enum TypeCreateAccountBanner {
     Plants
 }
 
-const images = {
-  [TypeCreateAccountBanner.People]: illustrationPeople,
-  [TypeCreateAccountBanner.Plants]: illustrationPlant,
+const bgStyles = {
+  [TypeCreateAccountBanner.People]: styles.withPeople,
+  [TypeCreateAccountBanner.Plants]: styles.withPlants,
```

Comment:
I feel I should make background images part of the style rather than a html `img`.  Better for accessibility etc.  Is that true?